app/assets/javascripts/application.js

```diff
@@ -13,7 +13,7 @@ $(function() {
 
     // $('el:bottom-offscreen') will return true if the element's
     // bottom box border is off the screen
-    $.expr.filters['bottom-offscreen'] = function(el) {
+    $.expr.pseudos['bottom-offscreen'] = function(el) {
```

Comment:
Why has this method changed? Was `filters` replaced?
Ah, I see that it was: https://github.com/jquery/jquery-migrate/blob/69a24416078538ca267c6182207611002cdb9908/src/jquery/core.js#L102-L103

Reply:
Yup, deprecated in jQuery 3.

Thanks for the review Leena ⭐ 🙇